specificatie/brp/gezag/gezagsrelatie-v1.yaml

```diff
@@ -33,6 +33,7 @@ components:
           burgerservicenummer: '999998894'
 
     TweehoofdigOuderlijkGezag:
```

Comment:
Als @fsamwel en @CathyDingemanse en de rest van het team ermee eens is, dan kan dit type al worden hernoemd naar GezamenlijkOuderlijkGezag. Dit is nl. de specificatie voor een interne api

Reply:
eens

Reply:
ook mee eens

Reply:
Aangepast.